app/models/miq_worker.rb

```diff
@@ -219,6 +219,10 @@ def worker_settings(options = {})
     self.class.fetch_worker_settings_from_server(miq_server, options)
   end
 
+  def heartbeat_file
+    ENV["WORKER_HEARTBEAT_FILE"] || Rails.root.join("tmp", "#{guid}.hb")
```

Comment:
Sorry @NickLaMuro, I misread you comment and though you were asking why you got pinged, lol. I maded the change to cleanup the heartbeat file create on prior runs.